src/Raven.Server/RavenServerStartup.cs

```diff
@@ -336,18 +336,19 @@ private static void MaybeSetExceptionStatusCode(HttpContext httpContext, ServerS
             }
 
             if (exception is LowMemoryException ||
+                exception is HighDirtyMemoryException || 
                 exception is OutOfMemoryException ||
                 exception is VoronUnrecoverableErrorException ||
-                exception is DiskFullException ||
-                exception is HighDirtyMemoryException)
+                exception is VoronErrorException ||
+                exception is QuotaException ||
+                exception is DiskFullException)
             {
                 response.StatusCode = (int)HttpStatusCode.ServiceUnavailable;
                 return;
             }
 
             if (exception is DocumentConflictException ||
                 exception is ConflictException ||
-                exception is ConcurrencyException ||
```

Comment:
why was it removed?

Reply:
Actually, it was a ReSharper warning.  `ConcurrencyException` is `ConflictException` so it always returns `false`